team.go

```diff
@@ -101,8 +102,15 @@ type TeamMembership struct {
 	User UserId `graphql:"user"`
 }
 
+type TeamMembershipConnection struct {
+	Nodes      []TeamMembership
+	PageInfo   PageInfo
+	TotalCount int
+}
+
 type TeamMembershipUserInput struct {
 	User UserIdentifierInput `json:"user"`
+	Role string              `json:"role"`
```

Comment:
Should we have an enum for the types of roles instead of string?

Reply:
We cannot because they currently support a string via the API.  Also i think they are going to let customers define their own roles so thats will require a string.